desafio-02/JonataMSilva/python/numeros_primos.py

```diff
@@ -0,0 +1,12 @@
+def encontrar_num_primo(numero):
+    divisor = 0
+    for i in range(1, numero+1):
+        if numero % i == 0:
+            divisor += 1
+    if divisor == 2:
+        return True
+    return False
+
+for primos in range(1, 10000):
```

Comment:
Neste trecho aqui quando definido "primos" in range, não necessariamente o número sera um primo pode ser ou não ser, então acredito que ficaria mais descritivo utilizar numero in range. faz sentido ?